### PR TITLE
Added my name "ian"

### PR DESCRIPTION
**Pull request by ianfridman**
This adds the file ianfridman because it describes a memeber of this fabulous class.

### CODE REVIEW

File: Participants/ianfridman [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name: Ian Fridman
```
Comment (el2731) [inline Participants/ianfridman:1]: Awesome!

**Review by el2731 — CHANGES_REQUESTED** [review]
Add a new empty line after line 3.

File: Participants/ianfridman (hunk 1 above)
Comment (ianfridman) [reply Participants/ianfridman:1]: Yes!

**Comment by ianfridman** [comment]
I changed it up! Please review again :)

**Review by el2731 — APPROVED** [review]
Looks good! Great job!

**Pull request merged**